joomla/joomla-cms
# Remove full stops from query explanation clauses

#### Summary of Changes

Some language strings in Smart Search are intended to be used as clauses within a constructed sentence.  Although they look like sentences in their own right, this is not how they are used and hence they should not end in full stops.

The sentence is assembled here: https://github.com/joomla/joomla-cms/blob/staging/components/com_finder/helpers/html/query.php

These language strings are not used anywhere else.
#### Testing Instructions
- Make sure you have Smart Search enabled and some content indexed.
- Make sure you have "Query explanation" set to "Yes" in a menu item or in the global options.
- Enter a search term (eg. "the").  The query explanation given will be something like this: "Assuming **the** is required., the following results were found.".   Note the full stop after "required".
- Apply this PR.
- Repeat the search and you should see: "Assuming **the** is required, the following results were found.".
- Similarly for the other clauses.
